build/flags.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -f "build/env.sh" ]; then
+    echo "$0 must be run from the root of the repository."
+    exit 2
+fi
+
+GO_ETHEREUM_FLAGS=""
+
+# set gitCommit when running from a Git checkout.
+if [ -f ".git/HEAD" ]; then
+    GO_ETHEREUM_FLAGS=$GO_ETHEREUM_FLAGS"-ldflags '-X main.gitCommit $(git rev-parse HEAD)'"
+fi
+
+: ${GO_OPENCL:="false"}
```

Comment:
```:``` is a null command, not entirely sure why it was needed here. It should just expand the values of its arguments. ```${GO_OPENCL:="false"}``` is the correct POSIX way of substituting the variable only if it's set AND not null: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02